### PR TITLE
Add a section to point to Lr API docs

### PR DESCRIPTION
**Pull request by shayanc1985**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -933,3 +935,15 @@ The typical workflow involves making a synchronous API call to the POST endpoint
...
 
 The API's are documented at [https://adobedocs.github.io/photoshop-api-docs/#api-Sensei-ImageCutout](https://adobedocs.github.io/photoshop-api-docs/#api-Sensei-ImageCutout)
+
+# Lightroom APIs
+
+The Adobe Lightroom APIs allow you to make Lightroom like automated edits to JPG image files.
+
+## General Workflow
+
+The typical workflow involves making a synchronous API call (specifying the hrefs to the input and the output assets) to the POST endpoint https://image.adobe.io/lrService/ for which the response will contain a link to an asynchronous job thats created. Making a synchronous GET API call to this link will return the status of the job.
```
Comment (noahhoffman) [inline README.md]: @shayanc1985 
"The typical workflow involves making an API POST call to the endpoint https://image.adobe.io/lrService/ for which the response will contain a link to check the status of the asynchronous job. Making a GET call to this link will return the status of the job and, eventually, the links to your generated output."

File: README.md [hunk 2]
```diff
@@ -933,3 +935,15 @@ The typical workflow involves making a synchronous API call to the POST endpoint
 ## How to use the API's
 
 The API's are documented at [https://adobedocs.github.io/photoshop-api-docs/#api-Sensei-ImageCutout](https://adobedocs.github.io/photoshop-api-docs/#api-Sensei-ImageCutout)
+
+# Lightroom APIs
+
+The Adobe Lightroom APIs allow you to make Lightroom like automated edits to JPG image files.
```
Comment (noahhoffman) [inline README.md]: @shayanc1985 
"The Adobe Lightroom APIs allow you to apply Lightroom-like, automated edits to JPG image files."

**Comment by noahhoffman** [comment]
👍

File: README.md (hunk 2 above)
Comment (shayanc1985) [reply README.md]: @noahhoffman I realized that i need to remove JPG and make it just "image files" since we support multiple input formats.

**Pull request merged**